index.html

```diff
@@ -1232,6 +1232,18 @@ <h3>Special Ability</h3>
                   <td>ShldVamp</td>
                   <td>Can steal energy from player ship shields (unconfirmed)</td>
                 </tr>
+                <tr>
+                  <td><code>0x1000</code></td>
+                  <td>ELITE_?</td>
```

Comment:
Damn, these aren't documented in mission-file-docs.txt yet? That feels like a bugreport   :confounded:

Reply:
I concur.

Reply:
Bug reported at https://artemis.forumchitchat.com/post/2-7-4-bug-reports-10388698